### ISSUE TITLE
[BUG] Use of mbstring.internal_encoding is deprecated (PHP 8.0)

### ISSUE THREAD

**Issue by lkppo** [simplepie/simplepie#701]
> **IMPORTANT:** If you are posting code or output, please make sure you wrap it with [fenced code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/) so that it is readable.

#### Description

PHP emits a warning about mbstring.internal_encoding deprecation. This INI option is deprecated since PHP 7.2 but the notice is emitted only since PHP 8.0.

#### Steps to Reproduce

Use the SimplePie_Misc::debug() method with mbstring loaded.  When mb_get_info check 'func_overload' the notice is emotted.

```
					case 'mbstring':
						$info .= '      Overloading: ' . mb_get_info('func_overload') . "\n";
```

#### Expected Behavior

Silent.

**Comment by jrfnl** [comment]
@lkppo I just tried and failed to reproduce this. Yes, the `func_overload` option is no longer supported in PHP 8.0, but I still didn't manage to get a deprecation notice when I ran `var_dump(mb_get_info('func_overload'));`. Also not sure what the reference to `mbstring.internal_encoding` is about. Again, yes that option has been deprecated since PHP 5.6, but that option is not referred to in the `debug()` method.

https://www.php.net/manual/en/function.mb-get-info.php
